docs/pull-requests.md

```diff
@@ -50,7 +50,7 @@ If there are specific changes within your pull request that you'd like to call y
 
 ### Create smaller pull requests
 
-Large pull requests are extremely painful to review. They also need to be kept up to date more frequently since they have a higher chance of creating conflicts that need to be resolved, and they make the commit history more difficult to understand after they are merged.
+Large pull requests are extremely painful to review. They also need to be kept up-to-date more frequently since they have a higher chance of creating conflicts that need to be resolved, and they make the commit history more difficult to understand after they are merged.
```

Comment:
Prosodically, it sounds like it ought to be "up to date". The internet is divided over this one, but grammatically I think you might be correct.